$OpenBSD: patch-midori_main_c,v 1.8 2011/05/17 21:03:10 landry Exp $
use path to our root ca list for ssl cert validation
--- midori/main.c.orig	Mon May 16 01:15:28 2011
+++ midori/main.c	Tue May 17 23:02:24 2011
@@ -1039,10 +1039,7 @@ midori_load_soup_session (gpointer settings)
     #if WEBKIT_CHECK_VERSION (1, 1, 14) && defined (HAVE_LIBSOUP_2_29_91)
     const gchar* certificate_files[] =
     {
-        "/etc/pki/tls/certs/ca-bundle.crt",
-        "/etc/ssl/certs/ca-certificates.crt",
-        "/etc/ssl/certs/ca-bundle.crt",
-        "/usr/local/share/certs/ca-root-nss.crt",
+        "/etc/ssl/cert.pem",
         NULL
     };
     guint i;
